templates/docs/examples/patterns/navigation/subnav.html

```diff
@@ -20,63 +20,63 @@
         <a href="#main-content">Jump to main content</a>
       </span>
       <ul class="p-navigation__items">
-        <li class="p-navigation__item p-subnav" id="link-1">
```

Comment:
We should keep existing p-subnav examples (for backwards compatibility) and add new examples for updated navigation.